# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../crm114/message-munging.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Message munging work around (see bug #406207).

Index: crm114/maillib.crm
===================================================================
--- ./maillib.crm	2009-10-20 11:12:34.000000000 +0200
+++ ./maillib.crm	2010-01-23 15:03:22.000000000 +0100
@@ -371,11 +371,13 @@
 	match (:found: :hd: :tail:) <nomultiline nocase> /^(:*:header:) (.*)/
 	alter (:tail:) /:*:tail: (:*:comment:)/
     }
-    alius
-    {
-	#    no such header.  make one.
-	call /:mungmail_add:/ [:*:header: (:*:comment:)]
-    }
+# Commented out by Milan Zamazal <pdm@debian.org> on 2007-01-28 as suggested on
+# the crm114 mailing list.  If this is wrong please let me know.
+#   alius
+#   {
+#       #    no such header.  make one.
+#       call /:mungmail_add:/ [:*:header: (:*:comment:)]
+#   }
 }
 return
 
